stability/stability-manual-android-tests/testscripts/Download_File_LongTime.feature

```diff
@@ -0,0 +1,6 @@
+Feature: stability-android
+ Scenario: Download_File_LongTime
+  When launch "downloadtest"
+    And I wait 15 seconds
+    And repeat to download resources from link "tizen-ivi_20150117.1_ivi-efi-x86_64-sdb.raw.bz2" for 1800 seconds
+  Then I should see "tizen-ivi_20150117.1_ivi-efi-x86_64-sdb.raw.bz2"
```

Comment:
How to make sure the "tizen-ivi_20150117.1_ivi-efi-x86_64-sdb.raw.bz2" you see is the only one you expected? Seems there's also a same text of link.